cypress/integration/computation_menu.spec.ts

```diff
@@ -0,0 +1,327 @@
+        });
+    });
+
+    context("District seats", () => {
+        yearsWithModernElectionSystem.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").wait(1000);
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const districtSeatValues = ["0", "125", "250", "325", "500"];
+                districtSeatValues.forEach((districtSeats) => {
+                    cy.get("#districtSeats").clear();
+                    cy.get("#districtSeats").type(districtSeats);
+                    cy.get("#districtSeats").should("have.value", districtSeats);
+                });
+            });
+        });
+    });
+
+    context("Area factor", () => {
+        yearsWithModernElectionSystem.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").wait(1000);
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const areaFactorValues = ["0", "0.75", "1.5", "2.25", "3"];
+                areaFactorValues.forEach((areaFactor) => {
+                    cy.get("#areaFactor").clear();
+                    cy.get("#areaFactor").type(areaFactor);
+                    cy.get("#areaFactor").should("have.value", areaFactor);
+                });
+            });
+        });
+    });
+
+    context("Modern combinations", () => {
+        const presentations = [
+            "Landsoversikt",
+            "Fylkesoversikt",
+            "Utjevningsmandater",
+            "Restkvotienter",
+            "Fylkesfordeling av mandater",
+            "Enkeltfylke",
+        ];
+        const algorithms = ["SAINTE_LAGUE", "D_HONDT", "LARGEST_FRACTION_HARE"];
+        const year = 2017;
+        const areaFactorValues = ["0", "3"];
+        const districtSeatValues = ["0", "500"];
+        const levelingSeatValues = ["0", "100"];
+        const districtThresholdValues = ["0", "15"];
+        const electionThresholdValues = ["0", "15"];
+        const firstDivisorValues = ["1", "5"];
+
+        presentations.forEach((presentation) => {
+            algorithms.forEach((algorithm) => {
+                areaFactorValues.forEach((areaFactor) => {
+                    districtSeatValues.forEach((districtSeats) => {
+                        levelingSeatValues.forEach((levelingSeats) => {
+                            districtThresholdValues.forEach((districtThreshold) => {
+                                electionThresholdValues.forEach((electionThreshold) => {
+                                    it(`Presentation ${presentation}, Year ${year}, Algorithm ${algorithm}, Area factor ${areaFactor}, District seats ${districtSeats}, Leveling seats ${levelingSeats}, District threshold ${districtThreshold}, Election threshold ${electionThreshold}`, () => {
+                                        cy.visit("").wait(2000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)